FeedStoreChallenge/InMemoryFeedStore.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  InMemoryFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by armand on 30/10/2020.
+//  Copyright © 2020 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public class InMemoryFeedStore: FeedStore {
+    private struct Cache {
+        let feedItems: [LocalFeedImage]
+        let timestamp: Date
+    }
+    
+    public init() {
+        
+    }
+    
+    private var localStore: [Cache] = []
```

Comment:
Since we only store a single cache value, it could be an `Optional` instead of an `Array`:

```suggestion
    private var cache: Cache?
```